tripleblindmarket/covid-safe-paths
# Add location tracking toggle and language picker to Settings screen

## Description

Per the Figma design, I added two new settings: a location on/off toggle, and a language picker.

The app wasn't equipped for displaying the state of a user who turned off tracking, so I had to make some additions to the LocationTracking.js screen as well to both show correct information and to update when the user changed the setting (without necessitating an app restart). For similar reasons, made a slight modification to Entry and Onboarding1 so returning users wouldn't be subjected to onboarding on every open.

The other changed files are new icons and translation strings.

## How to test

We need to make sure that the PR 
- Works as expected on high-resolution screens (it should, but would love to verify)
- Language picker works the same as the onboarding one on iOS

## Still to do
On startup, there's a small bug where the LocationTracking screen will show the status unknown icon/text for a brief moment before showing the correct status.

Also, I may need some help merging back into develop - didn't make friends with git the last time I tried. (UPDATE: Just rebased with develop and nothing seems to be on fire 👍)

## Screenshots
![Screenshot from 2020-04-17 08-28-17](https://user-images.githubusercontent.com/6233577/79575324-37016500-8087-11ea-8d36-e76892743d5c.png)
![Screenshot from 2020-04-17 08-28-30](https://user-images.githubusercontent.com/6233577/79575333-3963bf00-8087-11ea-86a2-894c5180eaa1.png)
![Screenshot from 2020-04-16 09-21-55](https://user-images.githubusercontent.com/6233577/79575400-58625100-8087-11ea-827d-f40ab5c5928e.png)
